backends/ebpf/target.h

```diff
@@ -239,10 +250,7 @@ class P4TCTarget : public KernelSamplesTarget {
         return "HOST"_cs;
     }
 
-    bool isPrimitiveByteAligned(int width) const {
-        return (width <= 8 || width <= 16 || (width > 24 && width <= 32) ||
-                (width > 56 && width <= 64));
-    }
+    bool isPrimitiveByteAligned(int width) const { return (TCisPrimitiveByteAligned(width)); }
```

Comment:
This is more TC-specific code in the eBPF back end. Please remove it here and use `TCisPrimitiveByteAligned` where appropriate.